charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "1.1.3"
+appVersion: 1.1.3
```

Comment:
Is there a reason to add the `v` elsewhere instead of just setting this to "v1.1.3"

Reply:
Although the spec says that `appVersion` needn't be a valid SemVer version, there are tools that either don't work correctly or error if it's not. The comment about quotes being recommended is also misleading as they're only relevant if you have a version that is a different data type.

Reply:
Additionally, for many reasons which I won't bother listing, sticking with a valid SemVer 2 version here will cause the least potential for disruption as tools and technologies move on.

Reply:
Fair enough